README.md

```diff
@@ -1121,13 +1110,13 @@ You should build and install libexpat and zlib.  You may use brew, macports, bui
 I recommend that you build and install CMake from source.
 
 [TOC](#TOC)
-<div id="5-3">
+<div id="PlatformMinGWMSYS">
 
-### 5.3 MinGW/msys2
+## MinGW/msys2
 
 Please note that the platform MinGW/msys2 32 is obsolete and superceded by MinGW/msys2 64.
 
-#### MinGW/msys2 64 bit
+### MinGW/msys2 64 bit
 Install: [https://repo.msys2.org/distrib/x86\_64/msys2-x86\_64-20200903.exe](https://repo.msys2.org/distrib/x86_64/msys2-x86_64-20200903.exe)
```

Comment:
Let's link to "latest" instead of specific dated version here: https://repo.msys2.org/distrib/msys2-x86_64-latest.exe

Or just link to the MSYS2 project webpage only: https://www.msys2.org/

Reply:
Fixed. I will actually go through the MSYS2 documentation when trying to fix the problems I am having with MSYS builds at #2090. If I find something else to change in the documentation, I will update it in the other PR.

Reply:
Thanks, looking good!